src/packages/admin-ui-components/src/input/component.tsx

```diff
@@ -30,6 +30,11 @@ export const Input = ({
 	const [value, setValue] = useState<string | undefined>(initialValue ?? '');
 	const debouncedValue = useDebounce<string | undefined>(value, 800); // debounce request for 800ms
 
+	useEffect(() => {
```

Comment:
Without this change, the new TextFilter does not empty its value when clicking on 'Empty FIlters'. Another solution would be we have an uncontrolled Input component that does not track state like this

Reply:
I think it should still be a controlled input, just not have its own state, e.g. use the filter to hold the state the way the rest of the filter components do.